__tests__/services/CleanupService.spec.ts

```diff
@@ -304,39 +310,61 @@ describe('CleanupService', () => {
             value: true,
           },
         },
-      }
+      };
       beforeEach(() => {
         when(global.browser.contextualIdentities.query)
           .calledWith({})
-          .mockResolvedValue([{cookieStoreId: 'firefox-container-1'}] as never);
+          .mockResolvedValue([
+            { cookieStoreId: 'firefox-container-1' },
+          ] as never);
         when(global.browser.cookies.getAllCookieStores)
           .calledWith()
-          .mockResolvedValue([{id: 'firefox-default'}] as never);
+          .mockResolvedValue([{ id: 'firefox-default' }] as never);
         when(global.browser.cookies.getAll)
-          .calledWith({storeId: 'firefox-default'})
+          .calledWith({ storeId: 'firefox-default' })
           .mockResolvedValue([
             mockCookie, // no list
             googleCookie, // greylist, opentab
             youtubeCookie, // whitelist
-            yahooCookie,  // no list
+            yahooCookie, // no list
             openTabCookie, // opentab
-            githubCookie,  // greylist
+            githubCookie, // greylist
           ] as never);
       });
 
       it('Regular clean, exclude open tabs.', async () => {
-        const ffResult = await cleanCookiesOperation(firefoxState, cleanupProperties);
-        expect(global.browser.contextualIdentities.query).not.toHaveBeenCalled();
-        expect(global.browser.extension.isAllowedIncognitoAccess).toHaveBeenCalledTimes(1);
-        expect(global.browser.cookies.getAllCookieStores).toHaveBeenCalledTimes(1);
-        expect(global.browser.cookies.getAll).toHaveBeenCalledWith({storeId: 'firefox-default'});
-        expect(global.browser.cookies.getAll).toHaveBeenCalledWith({storeId: 'default'});
+        const ffResult = await cleanCookiesOperation(
+          firefoxState,
+          cleanupProperties,
+        );
+        expect(
+          global.browser.contextualIdentities.query,
+        ).not.toHaveBeenCalled();
+        expect(
+          global.browser.extension.isAllowedIncognitoAccess,
+        ).toHaveBeenCalledTimes(1);
+        expect(global.browser.cookies.getAllCookieStores).toHaveBeenCalledTimes(
+          1,
+        );
```

Comment:
Interesting how eslint/prettier made this into separate line(s).

Reply:
Ack.